readme.md

```diff
@@ -10,14 +10,23 @@ Join us on [Bozeman Laravel Slack]
 - [Composer](https://getcomposer.org/download/)
 - [NodeJS](https://nodejs.org/en/)
 ## Getting Started
+### Using [Laravel Sail](https://laravel.com/docs/8.x/sail#installation)
 - Clone this repository on your machine and `cd` into it
-- Run `composer install`
```

Comment:
@dhildreth is there a reason for removing the composer install and .env steps in the Sail documentation?